sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_critical_result_inference_sync.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import datetime
+import os
+
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.radiologyinsights import RadiologyInsightsClient
+from azure.healthinsights.radiologyinsights import models
+
+"""
+FILE: sample_critical_result_inference_async.py
```

Comment:
Nit: this should match the actual file name. 

Also, no need to have `_sync` at the end of the sync samples and tests. you can just keep the suffix on the async ones.

Reply:
changed and fixed